v2/openapi/fb-unified-openapi.yaml

```diff
@@ -3800,6 +3802,8 @@ components:
             - $ref: '#/components/schemas/PublicBlockchainCapability'
             - $ref: '#/components/schemas/IbanCapability'
             - $ref: '#/components/schemas/SwiftCapability'
+            - $ref: '#/components/schemas/PeerAccountTransferCapability'
+            - $ref: '#/components/schemas/InternalTransferCapability'
```

Comment:
Are you sure?
We always assumed these kinds of transfer are to `accountId` as the dest address and it makes no sense creating new ones.